src/resources/parser/texture/dds.js

```diff
@@ -5,88 +5,17 @@ import {
-    PIXELFORMAT_R8_G8_B8_A8,
+    PIXELFORMAT_R8_G8_B8, PIXELFORMAT_R8_G8_B8_A8,
     PIXELFORMAT_RGBA32F,
     TEXHINT_ASSET
 } from '../../../graphics/constants.js';
 import { Texture } from '../../../graphics/texture.js';
 
-/* eslint-disable no-unused-vars */
-
-// Defined here:
-// https://docs.microsoft.com/en-us/windows/desktop/direct3ddds/dds-header
-const DDSD_CAPS = 0x1;            // Required in every .dds file.
-const DDSD_HEIGHT = 0x2;          // Required in every .dds file.
-const DDSD_WIDTH = 0x4;           // Required in every .dds file.
-const DDSD_PITCH = 0x8;           // Required when pitch is provided for an uncompressed texture.
-const DDSD_PIXELFORMAT = 0x1000;  // Required in every .dds file.
-const DDSD_MIPMAPCOUNT = 0x20000; // Required in a mipmapped texture.
-const DDSD_LINEARSIZE = 0x80000;  // Required when pitch is provided for a compressed texture.
-const DDSD_DEPTH = 0x800000;      // Required in a depth texture.
-
-const DDSCAPS2_CUBEMAP = 0x200;
-const DDSCAPS2_CUBEMAP_POSITIVEX = 0x400;
-const DDSCAPS2_CUBEMAP_NEGATIVEX = 0x800;
-const DDSCAPS2_CUBEMAP_POSITIVEY = 0x1000;
-const DDSCAPS2_CUBEMAP_NEGATIVEY = 0x2000;
-const DDSCAPS2_CUBEMAP_POSITIVEZ = 0x4000;
-const DDSCAPS2_CUBEMAP_NEGATIVEZ = 0x8000;
-
-const DDS_CUBEMAP_ALLFACES = DDSCAPS2_CUBEMAP |
-      DDSCAPS2_CUBEMAP_POSITIVEX | DDSCAPS2_CUBEMAP_NEGATIVEX |
-      DDSCAPS2_CUBEMAP_POSITIVEY | DDSCAPS2_CUBEMAP_NEGATIVEY |
-      DDSCAPS2_CUBEMAP_POSITIVEZ | DDSCAPS2_CUBEMAP_NEGATIVEZ;
-
-// Defined here:
-// https://docs.microsoft.com/en-us/windows/desktop/direct3ddds/dds-pixelformat
-const DDPF_ALPHAPIXELS = 0x1;
-const DDPF_ALPHA = 0x2;
-const DDPF_FOURCC = 0x4;
-const DDPF_RGB = 0x40;
-const DDPF_YUV = 0x200;
-const DDPF_LUMINANCE = 0x20000;
-
-/* eslint-enable no-unused-vars */
-
-// FourCC construction
-function makeFourCC(str) {
-    return str.charCodeAt(0) +
-           (str.charCodeAt(1) << 8) +
-           (str.charCodeAt(2) << 16) +
-           (str.charCodeAt(3) << 24);
-}
-
-const DDS_MAGIC = makeFourCC('DDS ');
-
-// Standard
-const FCC_DXT1 = makeFourCC('DXT1');
-const FCC_DXT5 = makeFourCC('DXT5');
-const FCC_DX10 = makeFourCC('DX10');
-const FCC_FP32 = 116; // RGBA32f
-
-// Non-standard
-const FCC_ETC1 = makeFourCC('ETC1');
-const FCC_PVRTC_2BPP_RGB_1 = makeFourCC('P231');
-const FCC_PVRTC_2BPP_RGBA_1 = makeFourCC('P241');
-const FCC_PVRTC_4BPP_RGB_1 = makeFourCC('P431');
-const FCC_PVRTC_4BPP_RGBA_1 = makeFourCC('P441');
-
-const fccToFormat = {};
-fccToFormat[FCC_FP32] = PIXELFORMAT_RGBA32F;
-fccToFormat[FCC_DXT1] = PIXELFORMAT_DXT1;
-fccToFormat[FCC_DXT5] = PIXELFORMAT_DXT5;
-fccToFormat[FCC_ETC1] = PIXELFORMAT_ETC1;
-fccToFormat[FCC_PVRTC_2BPP_RGB_1] = PIXELFORMAT_PVRTC_2BPP_RGB_1;
-fccToFormat[FCC_PVRTC_2BPP_RGBA_1] = PIXELFORMAT_PVRTC_2BPP_RGBA_1;
-fccToFormat[FCC_PVRTC_4BPP_RGB_1] = PIXELFORMAT_PVRTC_4BPP_RGB_1;
-fccToFormat[FCC_PVRTC_4BPP_RGBA_1] = PIXELFORMAT_PVRTC_4BPP_RGBA_1;
-
 /**
- * @private
```

Comment:
Just wondering why the private tag is being removed here. `DdsParser` isn't exported as public API. Does this class now appear in the API referecne docs?